content/pandas/concepts/dataframe/terms/replace/replace.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: '.replace()'
+Description: 'Returns DataFrame Object after values within DataFrame have been changed.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Data Structures'
+  - 'Dynamic Programming'
```

Comment:
```suggestion
  - 'Data Structures'
  - 'Dynamic Programming'
  - 'Functions'
  - 'Methods'
```

Since this entry pertains to a specific `DataFrame` method, let's add "Methods" to the list of `Tags`, for SEO purposes. (I think adding "Functions" might also be helpful, but you don't have to add that one.)